charts/prefect-worker/README.md.gotmpl

```diff
@@ -0,0 +1,21 @@
+{{ template "chart.header" . }}
+
+{{ template "chart.badgesSection" . }}
+
+{{ template "chart.description" . }}
+
+{{ template "chart.homepageLine" . }}
+
+## Getting Started Guide
+
+[Using Prefect Helm Chart for Prefect Workers](https://docs.prefect.io/latest/guides/deployment/helm-worker/)
```

Comment:
we can augment the base template that `helm-docs` uses by adding this `.gotmpl` template, and the `{{ template }}` blocks ive added here are (1) from the library and (2) should match our existing autogenerated blocks, in case we ever want to augment the existing readme format for the other charts